doc/meta.xml

```diff
@@ -137,12 +137,36 @@ hello-2.3  A program that produces a familiar, friendly greeting
 
   <varlistentry>
     <term><varname>license</varname></term>
-    <listitem><para>The license for the package. One from the
-    attribute set defined in <link
-    xlink:href="https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix">
-    <filename>nixpkgs/lib/licenses.nix</filename></link>.  Example:
-    <literal>stdenv.lib.licenses.gpl3</literal>. For details, see
-    <xref linkend='sec-meta-license'/>.</para></listitem>
+    <listitem>
+      <para>
+        The license, or licenses, for the package. One from the attribute set
+        defined in <link
+          xlink:href="https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix">
+          <filename>nixpkgs/lib/licenses.nix</filename></link>. At this moment
+        using both a list of licenses and a single license is valid. If the
+        license field is in the form of a list representation it means that
+        parts of the package are licensed differently.  Each license should
+        preferably be referenced by their attribute. The non-list attribute
+        value can also be a space delimited string representation of the
+        contained attribute names or spdxIds. The following are all valid
+        examples:
+        <itemizedlist>
+          <listitem><para>Single license referenced by attribute (preferred)
+              <literal>stdenv.lib.licenses.gpl3</literal>.
+          </para></listitem>
+          <listitem><para>Single license referenced by attribute name (frowned upon)
```

Comment:
Perhaps it is a bit unclear. I'm trying to point out that the string value is only valid because it matches a license shortName, which is the same as the attribute name.